core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -174,6 +179,23 @@ class QualificationAppInfo(
     res
   }
 
+  private def calculateNoExecsStageDurations(all: Seq[StageQualSummaryInfo]): Long = {
+    // If there are Execs not associated with any stage, then some of the Execs may not be
+    // supported on GPU.  We need to estimate the duration of these Execs and add it to the
+    // unsupportedTaskDur. We estimate the duration by taking the average of the unsupportedTaskDur
```

Comment:
I'm not sure I follow this estimation.   We are trying to give some penalty for execs that have transitions but don't map to a stage (ie we don't have a duration), correct?

I'm wondering if we are already calculating this with like either the stages with no execs or job overhead time.